src/backend/distributed/replication/multi_logical_replication.c

```diff
@@ -1791,10 +1787,8 @@ CreateSubscriptions(MultiConnection *sourceConnection,
 		 * The ALTER ROLE command should not propagate, so we temporarily
 		 * disable DDL propagation.
 		 */
-		SendCommandListToWorkerOutsideTransaction(
-			target->superuserConnection->hostname,
-			target->superuserConnection->port,
-			target->superuserConnection->user,
```

Comment:
I feel like we should `CloseConnection` for all connections of `logicalRepTargetList` at some point, once we are done with all `SendCommandListToWorkerInSeparateTransaction`. Do we already close at somepoint?

Reply:
Ok, we seem to have `CloseGroupedLogicalRepTargetsConnections` which already closes the connections, so no need to worry about that

Reply:
Updated.